runinterface

```diff
@@ -4,4 +4,5 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 ./manage.py migrate
 ./manage.py storage_setup
+./manage.py sync_vmck &
```

Comment:
I think it would be better to define another task (a duplicate of [this one](https://github.com/vmck/acs-deploy/blob/master/nomad_jobs/acs-interface.hcl#L116-L233)) that runs the `sync_vmck` process in a separate docker container. That way you can see the logs separately and start/stop each of them as needed.